TestApp/.eslintrc

```diff
@@ -9,6 +9,7 @@
     ],
     "extends": "airbnb",
     "rules": {
+        "global-require": 0,
```

Comment:
@dhei thoughts? https://eslint.org/docs/rules/global-require

Reply:
It's better to keep this rule and make exceptions individually by `/* eslint-disable global-require */` comments.